.swiftlint.yml

```diff
@@ -0,0 +1,155 @@
+opt_in_rules:
+- attributes
+- closure_end_indentation
+- closure_spacing
+- empty_count
+- explicit_init
+- fatal_error_message
+- file_header
+- first_where
+- implicit_return
```

Comment:
Please turn off `implicit_return`. We shouldn't omit the `return` in multi-line closures.